includes/wikia/Wikia.php

```diff
@@ -30,6 +30,8 @@
 $wgHooks['AjaxAddScript']            [] = 'Wikia::onAjaxAddScript';
 $wgHooks['TitleGetSquidURLs']        [] = 'Wikia::onTitleGetSquidURLs';
 $wgHooks['userCan']                  [] = 'Wikia::canEditInterfaceWhitelist';
+$wgHooks['getUserPermissionsErrors'] [] = 'Wikia::canMoveMediaWikiNS';
+$wgHooks['getUserPermissionsErrors'] [] = 'Wikia::canUndeleteMediaWikiNS';
```

Comment:
These two could be handled in one hook since the code of the hook functions is nearly identical.